src/main/java/com/ecommerce/ECommerceApp/exception/InvalidPhoneNumberException.java

```diff
@@ -0,0 +1,10 @@
+package com.ecommerce.ECommerceApp.exception;
+
+/**
+ * Throws exception when invalide phone number is provided.
+ */
+public class InvalidPhoneNumberException extends RuntimeException {
+    public InvalidPhoneNumberException(String message) {
+        super(message);
+    }
+}
```

Comment:
I introduced a custom exception class to allow for more clear and targeted error handling inside the program. This will help make things easier to debug and provide a more clear understanding of what happened. 

Source: https://www.baeldung.com/java-clean-code